### PR TITLE
XW-4302 | Use template lint fully

### PR DESCRIPTION
**Pull request by hakubo**
Changes:
- use dot reporter by default
- enable recommended rules for template linter
- disabled some that would be too much work now to fix

### CODE REVIEW

**Review by JamesHawking — APPROVED** [review]
LGTM

**Review by rybmat — APPROVED** [review]

File: app/templates/components/lightbox-image.hbs [hunk 1]
```diff
@@ -1,6 +1,7 @@
-{{#unless loadingError}}
-	<img class="current" src="{{imageSrc}}" style={{style}}>
-{{else}}
+{{#if loadingError}}
 	{{i18n 'app.media-lightbox-error'}}
+{{/if}}
+{{#unless loadingError}}
```
Comment (rogatty) [inline app/templates/components/lightbox-image.hbs]: How about `{{#if}}{{else}}{{/if}}`?

**Review by rogatty — APPROVED** [review]
Good stuff :+1:

File: app/templates/components/article-comment.hbs [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <div class='avatar'>
```
Comment (latata) [inline app/templates/components/article-comment.hbs:1]: Can it check if only double quotes are used for attribute values?

Comment (hakubo) [reply app/templates/components/article-comment.hbs:1]: it will be able in next version

**Pull request merged**